introduction.md

```diff
@@ -12,4 +12,6 @@ My favorite path to become a great software craftman is to learn the following t
 * C#
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?